examples/example.php

```diff
@@ -1,20 +1,34 @@
 <?php
 
-//require
-require_once '../../../autoload.php';
-require_once 'config.php';
+/**
+ * Teamleader example
+ *
+ * This Teamleader PHP Wrapper class connects to the Teamleader API.
+ *
+ * @author Jeroen Desloovere <info@jeroendesloovere.be>
+ */
 
-use \SumoCoders\Teamleader\Teamleader;
-use \SumoCoders\Teamleader\Crm\Contact;
-use \SumoCoders\Teamleader\Crm\Company;
-use \SumoCoders\Teamleader\Opportunities\Sale;
-use \SumoCoders\Teamleader\Opportunities\SaleLine;
+// add your own credentials in this file
+require_once __DIR__ . '/credentials.php';
+
+// required to load
+require_once __DIR__ . '/../src/Teamleader.php';
```

Comment:
loading vendor/autoload should be enough to include these classes. When using PSR0 or PSR4 autoloading, you should never use require_once statements.